eth/downloader/queue.go

```diff
@@ -385,6 +385,9 @@ func (q *queue) Results(block bool) []*fetchResult {
 		for _, tx := range result.Transactions {
 			size += common.StorageSize(tx.Size())
 		}
+		for _, withdrawal := range result.Withdrawals {
+			size += withdrawal.Size()
+		}
```

Comment:
Since withdrawals are constant size, perhaps put `Size` on `Withdrawals` instead, and do
```golang
size += common.StorageSize(result.Withdrawals.Size())
```
(yes, make the conversion to StorageSize here, not in the Size-function)

Then the change in `types` boils down to 
```golang
var withdrawalSize = int(reflect.TypeOf(Withdrawal{}).Size())

func (s Withdrawals) Size() int {
	return withdrawalSize * len(s)
}
```